submissions/JuliaVorotchenko/Popup-task/index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+              content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="main.css">
+        <link href="https://fonts.googleapis.com/css?family=Work+Sans:400,500,600,700" rel="stylesheet">
+
+        <title>HTML & CSS practice: Google-style Popup</title>
+</head>
+<body>
+    <!--Меню-->
+    <div class="nav-wrapper">
+        <div class="navigation">
+            <div class="nav-item text"><a href="#">Gmail</a></div>
+            <div class="nav-item text"><a href="#">Images</a></div>
+            <!--Popup-->
+            <div class="nav-item nav-item-icon grid"><a href="#" class="grid-link"><img src="img/grid.svg" alt="grid menu icon"></a>
+                <input class="grid-top" type="checkbox" id="applications" name="" value="">
```

Comment:
Then I hover `input #applications` and click many times, I see a popup appear and disappear. That is ok. That is the way suppose to be.

But, I use TAB, the popup appears automatically. And I can not control it with the `space` button on a keyboard.

![http://i.imgur.com/8c8CemL.png](http://i.imgur.com/8c8CemL.png)

As a user, which uses a keyboard, I expect the following behavior: 

1. On freshly downloaded page press TAB button several times, goal is to focus on `grid`.
![http://i.imgur.com/4iacuTu.png](http://i.imgur.com/4iacuTu.png)

2. If TAB is pressed another time, then focus jumps to the next `nav-item` 
![http://i.imgur.com/7GnXEN3.png](http://i.imgur.com/7GnXEN3.png)

3. While focusing on `grid` popup should appear after the `space` button is pressed, and a popup should disappear after `space` pressed again. Again and again.

To reach this goal, you should use `:checked` pseudoclass for `#applications` Or (for its label)

Reply:
@JuliaVorotchenko
There is one more bug:
while hovering on icons inside popup, those are jumping up and down 
https://youtu.be/EvMI_iYsLZM

Reply:
> https://youtu.be/EvMI_iYsLZM
thank you, I have corrected it

Reply:
About this space key issue I cant find solution.  If I hover on grid icon the space key works opening and closing popup. But if I focus on it  - there is a bug. Can you please explain what I have to do with #applications:checked?